turn.lua

```diff
@@ -693,6 +693,20 @@ function courseplay:turn(vehicle, dt, turnContext)
 	----------------------------------------------------------
 	--allowedToDrive = allowedToDrive and not courseplay:needToWaitForTools(vehicle)
 
+
+	if vehicle.cp.drivingMode:get() == DrivingModeSetting.DRIVING_MODE_AIDRIVER then
+		-- in AIDriver based modes do not call / use legacy collision code
+		if vehicle.cp.driver.collisionDetector then
```

Comment:
Can we wrap this section into a AIDriver:isInTraffic() function?

Reply:
Done.

Reply:
Ok, but now it is the same as detectCollision() except the info text is not set and speed is regulated. Is this what we want? Or could just detectCollision() return a bool?

Reply:
You are right. The difference will be the TRAFFIC message. I will use detectCollision() with return value self.allowedToDrive - OK?

Reply:
ok